test/image/compare_pixels_test.js

```diff
@@ -35,15 +40,34 @@ function runAll () {
          * More info:
          * https://github.com/plotly/plotly.js/issues/62
          *
-         * 40 test cases are removed:
+         * 41 test cases are removed:
          * - font-wishlist (1 test case)
          * - all gl2d (38)
-         * - gl2d_bunny-hull (1)
+         * - gl3d_bunny-hull (1)
+         * - polar_scatter (1)
          */
-        t.plan(files.length - 40);
+        var mocks = allMocks.filter(function(mock) {
+            return !(
+                mock === 'font-wishlist.json' ||
+                mock.indexOf('gl2d') !== -1 ||
+                mock === 'gl3d_bunny-hull.json' ||
+                mock === 'polar_scatter.json'
```

Comment:
Can we add a corresponding issue for these and if so could you link to it in the code here with a 
"See issue [issuelink]"